src/05-derivatives/01-introduction.md

```diff
@@ -0,0 +1,296 @@
+# Goals and scope
+
+Derivatives are outputs of common processing pipelines, capturing data and
+meta-data sufficient for a researcher to understand and (critically) reuse those
+outputs in subsequent processing. Standardizing derivatives is motivated by use
+cases where formalized machine-readable access to processed data enables higher
+level processing.
+
+## Storage of derived datasets
```

Comment:
Currently, the "common principles" page actually gives more detail / examples / alternatives for storage of derivative data than does the derivatives page. I wonder if the former should focus on definitions of the terms source / raw / derived, and the fundamental conditions of filesystem separation, but defer examples of the various filesystem arrangements to here?

Reply:
I'm actually thinking we might want to consider the reverse. Instead of separating out discussion of nested-dataset organization into two locations, perhaps it should all go in `common-principles`?

Reply:
Personally content for it to go in either direction.

Reply:
Finally taking the time to go through this. Slight preference for @effigies suggestion.